tasks/main.yml

```diff
@@ -422,6 +479,14 @@
     description: Microsoft SQL Server Tools
     baseurl: "{{ mssql_client_repository }}"
     gpgcheck: true
+  when: ansible_pkg_mgr in ["yum", "dnf"]
+
+- name: Configure the Microsoft SQL Server Tools repository # noqa command-instead-of-module
+  command: 'zypper addrepo -c -f --name "Microsoft SQL Server Tools" {{ mssql_client_repository }} packages-microsoft-com-prod'
+  when: ansible_pkg_mgr == "zypper"
+  register: __zypper_addrepo_output
```

Comment:
I see you use the variable with the same name `__zypper_addrepo_output` twice. I'd recommend using a different variable each time for easier debugging. Thanks